spec/features/signup_google_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.feature 'user signup' do
+
+  def stub_omniauth
+    OmniAuth.config.test_mode = true
```

Comment:
following code conventions
```
RSpec.feature 'Register new user', type: :feature do
  context 'Sign Up via Google' do
    background do
      OmniAuth.config.test_mode = true
      OmniAuth.config.mock_auth[:google_oauth2] = OmniAuth::AuthHash.new({
        info: {
          email: 'jesse@mountainmantechnologies.com',
          first_name: 'Jesse',
          last_name: 'Spevack'
        },
        credentials: {
          token: 'abcdefg12345',
          refresh_token: '12345abcdefg',
          expires_at: DateTime.now,
        }
      })
    end

    scenario 'with partial user details heads to a pre-registration form' do
      visit root_path
      click_link 'Sign Up' # if link is not present - feature fails, so no need to double-verify link existence
      click_link 'Create with Google' # again, no need to double-verify
      
      # ensure we are on the registration form with pre-populated fields
      # fill in all blank fields
      # submit the form

      # ENSURE (as the last step) we are signed in and appropriate flash message is visible
    end
  end
end
```

purpose of the test is to ensure something is working, if any of the required steps for the scenario is not working, entire scenario fails, so no need to perform several expectations, u really just need to test a flow and ensure the target result is good